src/main/java/cd/go/plugin/config/yaml/transforms/StageTransform.java

```diff
@@ -25,6 +25,8 @@
     private static final String JSON_STAGE_APPROVAL_TYPE_FIELD = "type";
     private static final String YAML_STAGE_APPROVAL_TYPE_FIELD = "type";
     private static final String JSON_STAGE_JOBS_FIELD = "jobs";
+    private static final String JSON_STAGE_APPROVAL_ALLOW_ONLY_ON_SUCCESS_FIELD = "allowOnlyOnSuccess";
+    private static final String YAML_STAGE_APPROVAL_ALLOW_ONLY_ON_SUCCESS_FIELD = "allowOnlyOnSuccess";
```

Comment:
Please make this field consistent with the style in the plugin by using snake case. I mean `allow_only_on_success`.